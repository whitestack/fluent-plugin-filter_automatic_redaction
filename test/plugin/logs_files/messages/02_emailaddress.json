[
  {
    "input_message": "Lorem Ipsum john.doe@whitestack.com Lorem Ipsum"
  },
  {
    "input_message": "Lorem Ipsum jane.doe@whitestack.com Lorem Ipsum"
  }
]