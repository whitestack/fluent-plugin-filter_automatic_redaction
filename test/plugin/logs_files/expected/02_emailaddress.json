[
  {
    "input_message": "Lorem Ipsum john.doe@whitestack.com Lorem Ipsum",
    "output_message": "Lorem Ipsum <EMAILADDRESS> Lorem Ipsum"
  },
  {
    "input_message": "Lorem Ipsum jane.doe@whitestack.com Lorem Ipsum",
    "output_message": "Lorem Ipsum <EMAILADDRESS> Lorem Ipsum"
  }
]